src/chocolatey/infrastructure.app/nuget/NugetList.cs

```diff
@@ -26,12 +31,23 @@ public static class NugetList
     {
         public static IEnumerable<IPackage> GetPackages(ChocolateyConfiguration configuration, ILogger nugetLogger)
         {
+            return SearchPackagesImpl(configuration, nugetLogger);
+        }
+
+
+        public static int GetCount(ChocolateyConfiguration configuration, ILogger nugetLogger)
+        {
+            return SearchPackagesImpl(configuration, nugetLogger).Count();

```

Comment:
If you're wondering why I put `Count()` here, it's because I need IQueryable's Count. Otherwise it pulls down all the results, and gets the count of that.